application-glossary-ui/src/main/resources/Glossary/Code/GlossarySheet.xml

```diff
@@ -52,7 +52,7 @@
   &lt;dl&gt;
     &lt;dt #if (!$editing &amp;&amp; $hasEdit)
         class="editableProperty"
-        data-property="$escapetool.xml($services.model.serialize($object.getProperty('title').reference))"
+        data-property="$escapetool.xml($services.model.serialize($object.getPropertyReference('title')))"

```

Comment:
Note that this causes the glossary application to depends on 12.10.11 (it currently only requires 12.10). I guess it's acceptable, but I'm not really in charge of this application, so I prefer letting someone else decide.

Reply:
Indeed, it's weird that a new API was added in a bugfix release but it's the case : https://github.com/xwiki/xwiki-platform/blob/78f0a96fbe41e7f5c6242b1386bf1e7255dd1e1e/xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/api/Object.java#L142

@jmiba Does it mean that the xobject doesn't have the "title" property set in some circumstances? That seems weird since that property comes from the `GlossaryClass `XClass so it should always be there from what I see.